grammar/IonText.g4.txt

```diff
@@ -10,6 +12,10 @@
 //        on UTF-16 code units (this is a flaw in ANTLR).
 //      - The grammar doesn't validate unpaired surrogate escapes in symbols or strings
 //        (e.g. "\udc00")
+//   * Symbol Tables nor valid Symbol Number ranges are defined.
```

Comment:
I'm not really sure what this one means. Can you give some examples?

Reply:
Discussed offline; an example is a symbol identifier that is out of range of the current symbol table.

The following is invalid:

```$ion_symbol_table::{symbols:["foo"]} $99```

While the following is valid:

```$ion_symbol_table::{symbols:["foo"]} $10```

Reply:
Yes. I tried the most subtle rewrite of the comment to better clarify. I think it helps.